gyb_syntax_support/StmtNodes.py

```diff
@@ -5,7 +5,7 @@
     # labeled-stmt -> label ':' stmt
     Node('LabeledStmt', name_for_diagnostics='labeled statement', kind='Stmt',
          children=[
-             Child('LabelName', kind='IdentifierToken', name_for_diagnostics='name'),
+             Child('LabelName', kind='IdentifierToken', name_for_diagnostics='label name'),
```

Comment:
Doesn't seem to be a test for this case?

Reply:
I don’t actually think we ever hit this right now because we only parse a labeled statement if there is a label. But I think it doesn’t hurt either.